dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java

```diff
@@ -231,7 +240,8 @@ private URL getRegisteredProviderUrl(final Invoker<?> originInvoker) {
                 .removeParameter(QOS_PORT)
                 .removeParameter(ACCEPT_FOREIGN_IP)
                 .removeParameter(VALIDATION_KEY)
-                .removeParameter(INTERFACES);
+                .removeParameter(INTERFACES)
+                .removeParameters(asyncKey);

```

Comment:
Hi, 
I saw your ISSUE, I think async should be registered to the registry so that we can see if the provider supports asynchronous on dubbo-ops.
IMO, the better way is that the Consumer side does not use the Provider-side async, which can be analogous to the timeout override strategy in Consumer side.